docs/prod-host-setup.md

```diff
@@ -46,6 +46,20 @@ for consuming and storing this data safely. We suggest using any upper-bounded
 forms of storage, such as fixed-size or ring buffers, programs like `journald`
 or `logrotate`, or redirecting to a named pipe.
 
+### Logging and performance
+
+Some host configurations can have an effect on the performance of the process
+as Firecracker will generate host kernel logs during normal operations.
+
+The most recent example of this was the addition of `console=ttyAMA0` host
+kernel command line argument on one of our testing setups which, because this
+particular console made writing the logs slower than the previous one,
+degraded the snapshot restore time from 3ms to 8.5ms on `aarch64`. This
+happened as creating the tap device for snapshot restore generated host kernel
+logs, which were very slow to write. We recommend adding `quiet loglevel=1` to
+the host kernel command line to limit the number of messages written to the
+serial console.
```

Comment:
This last phrase seems to belong inside the first paragraph.

Reply:
Done/